paddle/operators/detail/grpc_server.h

```diff
@@ -45,6 +45,8 @@ class AsyncGRPCServer final : public sendrecv::SendRecvService::Service {
   void WaitCond(int cond);
   void SetCond(int cond);
   void WaitClientGet(int count);
+  bool CondEqualTo(int cond);
+  void SubCond(int arg);
```

Comment:
SubCond is not used in this PR.

Reply:
Yep, this PR is also work in progress and this interface will be used in rpc_server while recv a `BatchBarrier` RPC call, I will implement this shortly.